### ISSUE TITLE
2,8-dihydroxyadenine (DHA) urinary crytals

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7543]
**Preferred term label:**
2,8-dihydroxyadenine (DHA) urinary stones
- Round, yellow-brown DHA urine crystals 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614723   PMID: 2227934 1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
